modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItRaftCommandLeftInLogUntilRestartTest.java

```diff
@@ -183,6 +185,10 @@ private void restartClusterWithNotAppliedCommands(
 
             assertTrue(IgniteTestUtils.waitForCondition(() -> appliedIndexNode0.get() == appliedIndexNode1.get(), 10_000));
 
+            RaftGroupService raftGroupService = table.internalTable().partitionRaftGroupService(0);
+
+            raftGroupService.peers().forEach(peer -> raftGroupService.snapshot(peer).join());
```

Comment:
I though, that we've agreed, that we will use willCompleteSuccessfully or similar instead of join, haven't we?

Reply:
Done.